### PR TITLE
Generate all release note categories with labels

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

- Learnings from #11422

If reviewers add the right labels to pull requests then the release note categories can be generated automatically.

There are two new labels:
Screen Shot 2023-08-23 at 11 09 51 am

#### What should we test?

- No test.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ changelog:
       exclude:
         labels:
           - dependencies
-    - title: Dependencies 📦
+          - feature toggled
+          - technical changes only
+    - title: "Experimental features for testing 😎"
```
Comment (dacook) [inline .github/release.yml:11]: For this one, which really means "behind a feature toggle", I take inspiration from [JB](https://github.com/openfoodfoundation/openfoodnetwork/releases/tag/v4.4.2) and suggest something like:

```suggestion
    - title: "Features under construction 🚧"
```

File: .github/release.yml [hunk 2]
```diff
@@ -6,6 +6,13 @@ changelog:
...
+        - feature toggled
+    - title: "Technical changes 🔧"
```
Comment (dacook) [inline .github/release.yml:13]: Also inspired by JB, I like this icon:
```suggestion
    - title: "Technical changes 🛠"
```

**Review by dacook — COMMENTED** [review]
Hooray, I've been looking forward to doing this!

I've noticed that different releasers often use different icons and titles, and was waiting months for the right opportunity to standardise these a bit more. This is the perfect opportunity!
We could open up the conversation to #delivery-circle but it's probably not worth it.

Of course, the releaser will still have the opportunity to reformat as needed, but having the same starting point will make things more consistent.

**Comment by dacook** [comment]
We could consider adding the icons to the labels also, but the board is already very busy-looking so I think better not to.

File: .github/release.yml (hunk 1 above)
Comment (mkllnk) [reply .github/release.yml:11]: > Features under construction 🚧

I like it. It fulfils the purpose better:

* Some instance managers want to know about updates here. This was first requested by Kirsten to know when she can test and what to tell customers. And for instances with the feature toggled on, these are user facing changes.
* Most instance managers should ignore this. Rachel pointed out that it's a nightmare when lesser involved instances start to use features without knowing the full impact and potential problems. So "under construction" conveys much better to stay clear unless you are a construction worker.

File: .github/release.yml (hunk 2 above)
Comment (mkllnk) [reply .github/release.yml]: Okay, I managed to insert the coloured version. There's not much in it, I think. :shrug:

Comment (dacook) [reply .github/release.yml]: Both versions were coloured for me, I guess they appear differently in different environments (perhaps something to consider more). This is mine:
Screen Shot 2023-08-23 at 3 14 04 pm

**Comment by dacook** [comment]
Spec failures can be ignored, they are flaky specs.

In fact this branch doesn't touch any tested code. We can include "[skip ci]" in commit messages to prevent CI from running in this case. (I have to admit I don't always remember though).

**Review by filipefurtad0 — APPROVED** [review]
This will indeed speed up and standardize the release process - great improvement.
:clap: :clap: :clap:

**Comment by filipefurtad0** [comment]
> In fact this branch doesn't touch any tested code. 

Yes, this is indeed the case - merging :+1

> We can include "[skip ci]" in commit messages to prevent CI from running in this case. (I have to admit I don't always remember though).

Ahh - nice, did not know about this one... :eyes:

**Comment by filipefurtad0** [comment]
> We can include "[skip ci]" in commit messages

Found some documentation here:
https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

Thanks so much for raising this @dacook!

**Comment by filipefurtad0** [comment]
I was trying this out, without creating a new tag manually (which is now part of the process), and am getting this error, when I click `Generate release notes`:

image

**Comment by dacook** [comment]
Fixed! https://github.com/openfoodfoundation/openfoodnetwork/commit/78585afc9ce1854f7fdc858924289f808bfb2602

Screen Shot 2023-08-29 at 4 02 53 pm

**Comment by mkllnk** [comment]
Ooops, thanks for the fix, @dacook.

**Pull request merged**